research/object_detection/dataset_tools/create_ava_tf_record.py

```diff
@@ -0,0 +1,457 @@
+# Copyright 2019 The MediaPipe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Modified by Kaushik Shivakumar for the AVA Actions Dataset
+# to work without MediaPipe, code started by Bryan Seybold.
+
+r"""Code to download and parse the AVA dataset for TensorFlow models.
+
```

Comment:
I think the documentation here tells how to use AVA as a tf.data.dataset, but this tool has been modified to produce tf records of media sequence (tf.train.sequenceexamples) right ? Can we update the documentation to reflect that ?

Reply:
Sure